sphinx-doc/module-hoomd-write.rst

```diff
@@ -24,5 +25,8 @@ hoomd.write
     :members: Burst, DCD, CustomWriter, GSD
     :show-inheritance:
 
+    .. autoclass:: HDF5Logger(trigger, filename, logger, mode="a")
+        :members:
```

Comment:
```suggestion
        :members: flush write
```
`flush` currently doesn't appear in the documentation. If I recall correctly, the internal action's methods are somehow automatically made available to the parent - right? Does this work to make flush appear in the sphinx docs? Or do we need to define a placeholder in `HDF5Logger`?